### PR TITLE
feat: log claims

### PR DESCRIPTION
**Pull request by hperl**
Sometimes there are issues with mapping the claims from a SSO provider to an identity trait. This change adds the claims that we get back to the event, so that we can better debug these issues.

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

**Review by aeneasr — APPROVED** [review]

**Review by zepatrik — DISMISSED** [review]
Only setting the keys should not leak any PII.

**Comment by aeneasr** [comment]
CI fails

File: selfservice/strategy/oidc/strategy.go [hunk 1]
```diff
@@ -447,11 +451,13 @@ func (s *Strategy) HandleCallback(w http.ResponseWriter, r *http.Request, ps htt
 		}
 	}
 
-	if err := claims.Validate(); err != nil {
+	if err = claims.Validate(); err != nil {
 		s.forwardError(w, r, req, s.handleError(w, r, req, pid, nil, err))
 		return
 	}
 
+	span.SetAttributes(attribute.StringSlice("claims", maps.Keys(claims.RawClaims)))
```
Comment (aeneasr) [inline selfservice/strategy/oidc/strategy.go:459]: Wouldn't this leak PII into the tracing system?

Comment (hperl) [reply selfservice/strategy/oidc/strategy.go:459]: No, this is just the keys, meaning the claim keys. The PII is just in the values, which we don't send.

**Pull request merged**